### PR TITLE
Cap3 fab5 23.1

### PR DESCRIPTION
**Pull request by Biossalus**
File Structure & Naming conventions are ready. 
Have a look and let us know if we are missing any Classes/Interfaces.

### CODE REVIEW

**Review by 1Fungi99 — CHANGES_REQUESTED** [review]
Need to add "node_modules" to git ignore

File: manager/src/main/java/inventory/manager/Services/IProductController.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package inventory.manager.Services;
+
+public interface IProductController {
```
Comment (AbigailSwigert) [inline manager/src/main/java/inventory/manager/Services/IProductController.java]: Is this supposed to be IProductService? There is already a ProductController and it is not an interface.

**Review by AbigailSwigert — CHANGES_REQUESTED** [review]
I believe the IProductController file has the wrong name. Other than that, looks great. This had made me realize that we need to add an admin table to our database schema, I think.

**Comment by Biossalus** [comment]
Updated branch to reflect the concerns addressed:
Added *.node_modules to .the .gitignore
Fixed the name on IProductController -> IProductService

Added more packages to the structure with their corresponding classes/interfaces
Added Dependencies to the pom.xml
Refactored Admin & Customer classes/interfaces into a single 'Account' class/interface

We will still need that Admin table in the DB.

**Review by PrettyScript — COMMENTED** [review]
I noticed that there is a "PaginationResult Class" I'm not sure if we would need that class, from what we learned I was under the impression that we would just extend "PagingAndSortingRepository" from the corresponding repository. We could always add/delete classes, this doesn't have to be set & stone.

**Review by AbigailSwigert — APPROVED** [review]

**Review by PrettyScript — APPROVED** [review]

**Review by 1Fungi99 — APPROVED** [review]

**Review by Dylan-Fellows — APPROVED** [review]
Everything looks good. Great work!

**Pull request merged**